### PR TITLE
[NEW APP] Add Lidarr

### PR DESCRIPTION
**Pull request by andrewbrereton**

### CODE REVIEW

File: apps/lidarr/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: "3.7"
+services:
+  sonarr:
```
Comment (meienberger) [inline apps/lidarr/docker-compose.yml:3]: change to lidarr

File: apps/lidarr/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      - TZ=${TZ}
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ${APP_DATA_DIR}/data:/config
+      - ${ROOT_FOLDER_HOST}/media:/media
+    ports:
+      - ${APP_PORT}:8686
+    restart: unless-stopped
+    networks:
+      - tipi_main_network
```
Comment (meienberger) [inline apps/lidarr/docker-compose.yml:18]: Add the traefik labels to make it exposable 

```suggestion
      - tipi_main_network
    labels:
      traefik.enable: ${APP_EXPOSED}
      traefik.http.routers.lidarr.rule: Host(`${APP_DOMAIN}`)
      traefik.http.routers.lidarr.entrypoints: websecure
      traefik.http.routers.lidarr.service: lidarr
      traefik.http.routers.lidarr.tls.certresolver: myresolver
      traefik.http.services.lidarr.loadbalancer.server.port: 8686
```

File: apps/lidarr/config.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "../schema.json",
+  "name": "Lidarr",
+  "available": true,
```
Comment (meienberger) [inline apps/lidarr/config.json:4]:
```suggestion
  "available": true,
  "exposable": true,
```

**Review by meienberger — CHANGES_REQUESTED** [review]

**Comment by meienberger** [comment]
Thank you very much for your pull request! I requested a few changes and we are ready to go

**Comment by andrewbrereton** [comment]
Changes have been made. Requested a re-review.

**Comment by meienberger** [comment]
Thank you 👍

**Pull request merged**